src/main/java/seedu/address/logic/commands/CloseClaimCommand.java

```diff
@@ -86,4 +86,21 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(String.format(e.getMessage(), insuranceId, Messages.format(clientToCloseClaim)));
         }
     }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof CloseClaimCommand)) {
```

Comment:
lets keep it consistent with AddClaimCommand's equals method
`if (other instanceof CloseClaimCommand) { return ... }
return false`

Reply:
Good spot, thanks 🙂 will make the changes